pkg/sql/feature_derivation.go

```diff
@@ -400,5 +401,16 @@ func InferFeatureColumns(trainIR *ir.TrainClause) error {
 			}
 		}
 	}
+	// Write back to client information about feature derivation result
+	if wr != nil {
+		for target, fclist := range trainIR.Features {
+			for _, fc := range fclist {
+				for _, fm := range fc.GetFieldMeta() {
+					wr.Write(fmt.Sprintf("Using column (%s) in feature column (%T) as model construct param (%s)", fm.Name, fc, target))
+				}
+			}
+		}
+		wr.Write("\n")
```

Comment:
What's the '\n' for?

Reply:
To separate feature derivation logs and training logs.